lib/UserInterface/textinput/Callbacks.h

```diff
@@ -30,7 +30,7 @@ namespace textinput {
                           EditorRange& R /*out*/,
                           std::vector<std::string>& DisplayCompletions /*out*/)
     = 0;
-    virtual ~TabCompletion();
+    virtual ~TabCompletion() {}
```

Comment:
Why did you change this? This way the vtable symbols will pollute all .o files which includes Callbacks.h instead of Editor.o.